templates/careers/base.html

```diff
@@ -2,7 +2,6 @@
 
 {% block extra_metatags %}
   <link rel="alternate" type="application/rss+xml"  href="/careers/feed" title="All job roles | Canonical Careers">
-  <script src="{{ versioned_static('js/modules/fuse/fuse.js') }}"></script>
```

Comment:
Not sure I understand why this was moved to base_index, won't that mean this script will be loaded on pages that don't use fuse?